src/app.module.ts

```diff
@@ -26,7 +26,7 @@ import { ServeStaticModule } from '@nestjs/serve-static';
       rootPath: join(__dirname, '..', 'static'),
     }),
     HttpModule,
-    MongooseModule.forRoot('mongodb://db:27017/logDatabase'),
+    MongooseModule.forRoot('mongodb://localhost:27017/logDatabase'),
```

Comment:
rein aus interesse: bleibt das hardcoded `localhost` oder wird das irgendwann konfigurierbar gemacht? letzteres wäre natürlich cooler :)

Reply:
Das ist schon in Arbeit :)